### DISCUSSION TITLE
How to display or set Virtual-Database-Service logs?

### DISCUSSION THREAD

**Discussion by LuanShiZhiJian** [apache/skywalking#11719] [Q&A]
![image](https://github.com/apache/skywalking/assets/41558924/6631456d-1cd5-4868-9a73-ab340226e697)
![image](https://github.com/apache/skywalking/assets/41558924/8b1d92e7-5353-42c7-86b6-ae3ac7098a6c)

**Comment by LuanShiZhiJian** [comment]
If these services have logs, will they be automatically displayed?

**Comment by wu-sheng** [comment]
Service has logs, virtual service(DB) is conjectured only. No log belongs to it.

**Reply by LuanShiZhiJian** [reply]
Are all virtual services the same?

**Reply by wu-sheng** [reply]
Yes. Virtual service is a general concept. MQ, DB and Cache are the same. Just different layers.
